cloudbuild/prod/cloudbuild.yaml

```diff
@@ -18,20 +18,10 @@ steps:
       - --keyring=logflare-prod-keyring-us-central1
       - --key=logflare-prod-secrets-key
   - name: "gcr.io/cloud-builders/docker"
+    entrypoint: 'sh'
     args:
       [
-        "build",
-        "--build-arg",
-        "MAGIC_COOKIE",
```

Comment:
We can pull out all of the MAGIC_COOKiE stuff as well. See [here](https://hexdocs.pm/mix/1.14/Mix.Tasks.Release.html)
We just need to set the `RELEASE_COOKIE` in `.env.*` and they will connect.

Reply:
right now the MAGIC_COOKIE build arg is not even being used so we can remove it safely, the cluster is likely using the `release/COOKIE` file.

Reply:
I had a look at the staging cloudbuild.yaml we can remove MAGIC_COOKIE_PREFIX as well (and the MAGIC_COOKIE in the staging env)

Reply:
will remove